api.proto

```diff
@@ -163,3 +163,16 @@ service API {
     // code is returned.
     rpc Publish(PublishRequest) returns (PublishResponse) {}
 }
+
+// ActivityOp represents an activity stream operation.
+enum ActivityOp {
+    CREATE_PARTITION = 0;
+    DELETE_STREAM    = 1;
+}
+
+// ActivityStreamEvent represents an entry into the activity stream.
+message ActivityStreamEvent {
```

Comment:
Part of me wonders if we shouldn't follow the same type of pattern we follow in other places, e.g. like how we handle Raft operations (https://github.com/liftbridge-io/liftbridge/blob/master/server/proto/internal.proto#L9-L25).

Reply:
That would make sense. However, could there be a way to not have to copy and paste the Protobuf contents from one file to the other? We could maybe use a common Protobuf file that could be stored in the liftbridge-api project and included internally and from liftbridge itself?

Reply:
Sorry, I didn't mean to actually _use_ the `Op` definitions from Liftbridge, but rather use a similar pattern here. Something like:

```protobuf
enum ActivityOp {
    CREATE_PARTITION = 0;
    DELETE_STREAM    = 1;
}

message CreatePartitionOp {
    string stream   = 1;
    int32 partition = 2;
}

message DeleteStreamOp {
    string stream = 1;
}

message ActivityStreamEvent {
    ActivityOp op                       = 1;
    CreatePartitionOp createPartitionOp = 2;
    DeleteStreamOp deleteStreamOp       = 3;
}
```

Basically, rather than putting all of the parameters for the various ops in a single `message`, we have separate ones where `op` indicates which one is used. Make sense?

Reply:
Yes, that is also what I had in mind. :) I was just wondering if we could avoid copy-and-pasting some of those definitions, but I suppose that there is no way around it since we are not necessarily added all Raft events to the event steam but only a subset.

I'll implement that.

Reply:
Should be fixed in 098972d.